akka-actor/src/main/scala/akka/pattern/Patterns.scala

```diff
@@ -22,7 +22,8 @@ import scala.concurrent.ExecutionContext
 object Patterns {
   import akka.actor.ActorRef
   import akka.japi
-  import akka.pattern.{ after ⇒ scalaAfter, ask ⇒ scalaAsk, gracefulStop ⇒ scalaGracefulStop, pipe ⇒ scalaPipe }
+  import akka.pattern.{ after ⇒ scalaAfter, ask ⇒ scalaAsk, gracefulStop ⇒ scalaGracefulStop,
+    pipe ⇒ scalaPipe, retry ⇒ scalaRetry}
```

Comment:
this will fail the build -- please compile the file before submitting the PR; (Space before } needed)